### ISSUE TITLE
Improve task execution output

### ISSUE THREAD

**Issue by lexmag** [lexmag/dicon#3]

**Comment by ericmj** [comment]
Can you elaborate on this?

**Comment by whatyouhide** [comment]
We should likely improve the output so that for example a prefix with the host is printed before each message is printed on the shell:

```sh
$ mix dicon.control stop
==> Connected to example.com
<example.com> ok
```

or something along those lines. We should also generally give more feedback when running tasks.

**Comment by lexmag** [comment]
Right, exactly what @whatyouhide said, also I think we should replace progress-bar with spinner.

**Comment by whatyouhide** [comment]
Well, or implement the progress bar correctly so that it doesn't break when terminal is resized :P I like the idea of a progress bar, the only thing we should consider is how to deal with progress bar plus parallel upload of tars if we ever do them.

**Comment by ericmj** [comment]
I think printing the host is only interesting if we run commands in parallel on different hosts. Otherwise you get the host from "Connected to example.com" anyway.

**Comment by whatyouhide** [comment]
@ericmj the idea is definitely to start running commands in parallel :)
